src/vs/workbench/contrib/inlineChat/browser/inlineChatNotebook.ts

```diff
@@ -58,7 +58,8 @@ export class InlineChatNotebookContribution {
 					return fallback;
 				}
 
-				throw illegalState('Expected notebook editor');
+				// throw illegalState('Expected notebook editor');
+				return `<notebook>${editor.getId()}#${uri}`;
```

Comment:
@rebornix I'm not sure why this code path is hit with the new diff editor.
Any idea why or how I can track this.